test/extended/openstack/loadbalancers.go

```diff
@@ -382,6 +383,121 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack][lb][Serial] The O
+		deployment, err := clientSet.AppsV1().Deployments(oc.Namespace()).Create(context.TODO(),
+			testDeployment, metav1.CreateOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+		err = e2edeployment.WaitForDeploymentComplete(clientSet, deployment)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		g.By(fmt.Sprintf("Creating Openshift LoadBalancer Service with loadBalancerSourceRanges: '%s'", drop_sourcerange))
+		jig := e2eservice.NewTestJig(clientSet, oc.Namespace(), svcName)
+		jig.Labels = labels
+		svc, err := jig.CreateLoadBalancerService(loadBalancerServiceTimeout, func(svc *v1.Service) {
+			svc.Spec.Ports = []v1.ServicePort{{Protocol: v1.ProtocolUDP, Port: svcPort, TargetPort: intstr.FromInt(8081)}}
+			svc.Spec.Selector = labels
+			svc.Spec.LoadBalancerSourceRanges = []string{drop_sourcerange}
+		})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		g.By("Checks from openshift perspective")
+		loadBalancerId := svc.GetAnnotations()["loadbalancer.openstack.org/load-balancer-id"]
+		o.Expect(loadBalancerId).ShouldNot(o.BeEmpty(), "load-balancer-id annotation missing")
+		e2e.Logf("LB ID: %s", loadBalancerId)
+		o.Expect(svc.Spec.LoadBalancerSourceRanges).Should(o.Equal([]string{drop_sourcerange}))
+		svcIp := svc.Status.LoadBalancer.Ingress[0].IP
+		o.Expect(svcIp).ShouldNot(o.BeEmpty(), "FIP missing on svc Status")
+		e2e.Logf("Service IP: %s", svcIp)
+
+		g.By("Checks from openstack perspective")
+		allPages, err := octavialisteners.List(loadBalancerClient, octavialisteners.ListOpts{LoadbalancerID: loadBalancerId}).AllPages()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		listeners, err := octavialisteners.ExtractListeners(allPages)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		// Check there is only one listener in the LB
+		o.Expect(listeners).Should(o.HaveLen(1), "More than one listener found for LB '%s': %v", loadBalancerId, listeners)
+		listenerId := listeners[0].ID
+		e2e.Logf("LB listener ID: %s", listenerId)
+
+		// Check allowed_cidrs in the LB listener matches with the svc spec
+		slicesEqual := stringSlicesEqual(listeners[0].AllowedCIDRs, []string{drop_sourcerange})
+		o.Expect(slicesEqual).Should(o.BeTrue(), "Unexpected allowed_cidrs in Openstack LoadBalancer Listener '%s'", listenerId)
+		e2e.Logf("Expected allowed_cidrs '%v' in Openstack LoadBalancer Listener '%s'", listeners[0].AllowedCIDRs, listenerId)
+
+		// Check there is no connectivity
+		g.By("Connectivity checks - there should be no connectivity")
+		var connNumber int = 5
+		g.By(fmt.Sprintf("accessing the service %d times from outside and storing the name of the pods answering", connNumber))
+		results := make(map[string]int)
+		for i := 0; i < connNumber; i++ {
+			// https://github.com/kubernetes/kubernetes/blob/master/test/images/agnhost/README.md#netexec
+			podName, err := getPodNameThroughUdpLb(svcIp, fmt.Sprintf("%d", svcPort), "hostname")
+			if err != nil {
+				e2e.Logf("Error detected while accessing the LoadBalancer service on try %d: %q", i, err)
```

Comment:
maybe we can re-phrase this as this is not an error but an expected behaviour?

Reply:
Done